crates/mev-share-rpc-api/src/mev.rs

```diff
@@ -16,5 +16,5 @@ pub trait MevApi {
     /// Similar to `mev_sendBundle` but instead of submitting a bundle to the relay, it returns a
     /// simulation result. Only fully matched bundles can be simulated.
     #[method(name = "simBundle")]
-    async fn sim_bundle(&self, request: SendBundleRequest) -> RpcResult<()>;
+    async fn sim_bundle(&self, bundle: SendBundleRequest, sim_options: SimBundleOptions) -> RpcResult<SimBundleResponse>;
```

Comment:
are these documented anywhere?

couldn't find it here

https://docs.flashbots.net/flashbots-auction/searchers/advanced/rpc-endpoint#mev_simbundle

Reply:
I don't think these are documented, I pulled them from the reference ts client implementation https://github.com/flashbots/mev-share-client-ts/blob/main/src/client.ts#L195

Reply:
Added an issue in the docs to get them added https://github.com/flashbots/flashbots-docs/issues/403

Reply:
Adding it now